.github/actionlint.yml

```diff
@@ -1,3 +1,13 @@
 self-hosted-runner:
   labels:
     - qemu-host
```

Comment:
this doesn't mean it will be run on qemu-host, it's a lint setting that allows the string to appear

can remove if you want, but no operational reason to